lib/features/modeling/behavior/GroupBehavior.js

```diff
@@ -24,26 +24,14 @@ var HIGH_PRIORITY = 2000;
  */
 export default function GroupBehavior(
     bpmnFactory,
-    canvas,
+    bpmnjs,
     elementRegistry,
     eventBus,
     injector,
     moddleCopy
 ) {
   injector.invoke(CommandInterceptor, this);
 
-  /**
-   * Gets process definitions
-   *
-   * @return {ModdleElement} definitions
-   */
-  function getDefinitions() {
-    var rootElement = canvas.getRootElement(),
```

Comment:
Why not simply change this method with a call to `bpmnjs.getDefinitions()`?

Keeps the "how to get the definitions" in a single place.

Reply:
The method just felt unnecessary when we are using the viewer API directly. We do use them directly in other parts of our code base, like the [RootElementReferenceBehavior](https://github.com/bpmn-io/bpmn-js/blob/9e313273353ed1530958a9ee9d54368cd4d7581c/lib/features/modeling/behavior/RootElementReferenceBehavior.js#L49)

Reply:
Works for me!